deepmd/fit/ener.py

```diff
@@ -722,6 +729,9 @@ def init_variables(
         self.fitting_net_variables = get_fitting_net_variables_from_graph_def(
             graph_def, suffix=suffix
         )
+        if isinstance(self.descrpt, deepmd.descriptor.DescrptSeAtten):
+            type_embedding = get_tensor_by_name_from_graph(graph, 't_typeebd')
+            self.compress_type_embedding = tf.convert_to_tensor(type_embedding)
```

Comment:
It's more straightforward to do it directly in the type embedding class (`TypeEmbed.init_variables`) than doing it in the fitting network.
Actually, it's not directly related to the fitting network.